autoPyTorch/datasets/time_series_dataset.py

```diff
@@ -1,174 +1,1267 @@
+            else:
+                seasonality_h_value = int(
+                    np.round((n_prediction_steps // int(freq_value) + 1) * freq_value)
+                )
+
+                while minimal_seq_length < (num_splits - 1) * seasonality_h_value:
+                    if num_splits <= 2:
+                        break
+                    num_splits -= 1
+                if resampling_strategy_args is None:
+                    resampling_strategy_args = {'num_splits': num_splits}
+                else:
+                    resampling_strategy_args.update({'num_splits': num_splits})
+
+        num_seqs = len(sequence_lengths)
+
+        if resampling_strategy_args is not None and "n_repeats" in resampling_strategy_args:
+            n_repeats = resampling_strategy_args["n_repeats"]
+        else:
+            n_repeats = None
+        if (num_seqs < 100 and minimal_seq_length > 10 * n_prediction_steps) or \
+                minimal_seq_length > 50 * n_prediction_steps:
+            if n_repeats is None:
+                if num_seqs < 100:
+                    n_repeats = int(np.ceil(100.0 / num_seqs))
+                else:
+                    n_repeats = int(np.round(minimal_seq_length / (50 * n_prediction_steps)))
+
+        if n_repeats is None:
+            n_repeats = 1
+        if resampling_strategy == CrossValTypes.time_series_cross_validation:
+            n_repeats = min(n_repeats, minimal_seq_length // (5 * n_prediction_steps * num_splits))
+        elif resampling_strategy == CrossValTypes.time_series_ts_cross_validation:
+            seasonality_h_value = int(np.round(
+                (n_prediction_steps // int(freq_value) + 1) * freq_value)
+            )
+            while minimal_seq_length // 5 < (num_splits - 1) * n_repeats * seasonality_h_value - n_prediction_steps:
+                n_repeats -= 1
+
+        elif resampling_strategy == HoldoutValTypes.time_series_hold_out_validation:
+            n_repeats = min(n_repeats, minimal_seq_length // (5 * n_prediction_steps) - 1)
+
+        else:
+            n_repeats = 1
+
+        n_repeats = max(n_repeats, 1)
+
+        if n_repeats is None:
+            n_repeats = 1
+
+        if resampling_strategy_args is None:
+            resampling_strategy_args = {'n_repeats': n_repeats}
+        else:
+            resampling_strategy_args.update({'n_repeats': n_repeats})
+        return resampling_strategy, resampling_strategy_args
+
+    def create_cross_val_splits(
+            self,
+            cross_val_type: CrossValTypes,
+            num_splits: int,
+            n_repeats: int = 1,
+    ) -> List[Tuple[Union[List[int], np.ndarray], Union[List[int], np.ndarray]]]:
+        """
+        This function creates the cross validation split for the given task.
+
+        It is done once per dataset to have comparable results among pipelines
+        Args:
+            cross_val_type (CrossValTypes):
+            num_splits (int): number of splits to be created
```

Comment:
could you write the description in a new line with indentation? I think it messes with the documentation builder. [example](https://github.com/dengdifan/Auto-PyTorch/blob/4898ca5ef492b465ad1331df2b380b10be070087/autoPyTorch/data/base_feature_validator.py#L69)